index.html

```diff
@@ -0,0 +1,218 @@
+                <li class="main-program-activity"><i class="fa-solid fa-person-chalkboard"></i><h3>CC Exhibitions</h3><p>Appreciate various creations applying the latest frond-end frameworks.</p></li>
+                <li class="main-program-activity"><i class="fa-solid fa-user-tie"></i><h3>Forums</h3><p>Have the time to share your thoughts and opinions with experts of each topic.</p></li>
+                <li class="main-program-activity"><i class="fa-solid fa-network-wired"></i><h3>Workshops</h3><p>Try to create your own project using open source tools.</p></li>
+                <li class="main-program-activity"><i class="fa-solid fa-user-group"></i><h3>Networking</h3><p>Get opportunities to network with affiliates around the world.</p></li>
+            </ul>
+            
+            <div id="see-more"><a href="#speakers" id="see-more-link">CHECK THE COMPLETE PROGRAM HERE</a></div>
+        </div>
+    </section>
+    <!--end of event program section-->
+
+    <!--featured speakers section-->
+    <section id="featured-speakers">
+        <div id="speakers-container">
+            <h2 id="speaker-title">Featured speakers</h2>
+            <hr>
+            <ul id="speakers">
+                <!-- <li class="speaker-info">
+                    <ul>
+                        <li><img src="/images-capstone/speaker8.jpg" alt="speaker 8" /></li>
+                        <li>
+                            <div class="speaker-name">Yochai Benkler</div>
+                            <div class="speaker-cv">Lorem ipsum dolor sit amet consectetur.</div>
+                            <div class="speaker-data">
+                                <p>Lorem ipsum, dolor sit amet consectetur.
+                                Eos aspernatur, nesciunt dolores, officia, eum quae at illum error
+                                labore earum nihil harum enim unde deleniti delectus fuga iste quibusdam voluptatum.</p>
+                            </div>
+                        </li>
+                    </ul>
+                </li>
+                <li class="speaker-info">
+                    <ul>
+                        <li><img src="/images-capstone/speaker1.jpg" alt="speaker 8" /></li>
+                        <li>
+                            <div class="speaker-name">Yochai Benkler</div>
+                            <div class="speaker-cv">Lorem ipsum dolor sit amet consectetur.</div>
+                            <div class="speaker-data">
+                                <p>Lorem ipsum, dolor sit amet consectetur.
+                                Eos aspernatur, nesciunt dolores, officia, eum quae at illum error
+                                labore earum nihil harum enim unde deleniti delectus fuga iste quibusdam voluptatum.</p>
+                            </div>
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.